pkg/rest/server_test.go

```diff
@@ -3,87 +3,72 @@
 
 package rest
 
-import (
-	"context"
-	"fmt"
-	"math/big"
-	"testing"
-
-	"github.com/pkg/errors"
-
-	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/phayes/freeport"
-	"github.com/tellor-io/telliot/pkg/config"
-	"github.com/tellor-io/telliot/pkg/db"
-	"github.com/tellor-io/telliot/pkg/testutil"
-)
-
-func TestServer(t *testing.T) {
-	cfg := config.OpenTestConfig(t)
-	cfg.ServerWhitelist = []string{"0x053b09e98ede40997546e8bb812cd838f18bb146"}
-
-	DB, cleanup := db.OpenTestDB(t)
-	defer t.Cleanup(cleanup)
-
-	balInt := big.NewInt(350000)
-	err := DB.Put(db.BalanceKey, []byte(hexutil.EncodeBig(balInt)))
-	testutil.Ok(t, err)
-
-	port, err := freeport.GetFreePort()
-	testutil.Ok(t, err)
-
-	proxy, err := db.OpenLocalProxy(DB)
+// func TestServer(t *testing.T) {
```

Comment:
I am not sure what these tests actually tests so disabling for now until we revisit all tests.